core/src/smartcontracts/isi/permissions/judge.rs

```diff
@@ -467,7 +467,7 @@ pub mod builder {
         #[inline]
         fn new<V: IsAllowed<Operation = O> + Send + Sync + 'static>(validator: V) -> Self {
             Self {
-                validators: vec![Box::new(validator)],
+                validators: vec![validator.boxed()],
```

Comment:
```suggestion
                validators: vec![validator.box()],
```

Reply:
Sorry, not sure I follow. Do you propose renaming `boxed` to `box`?

Reply:
Yes. Minor point, but `box` can be both treated as a noun (`box` containing this object) and a verb (put this thing into a `Box`).

Reply:
Just realized `box` is a keyword. What's our stance on `r#`-escaped identifiers? Alternatively this could be called `into_boxed()`

Reply:
Boxed is fine then.